frontend/src/scenes/session-recordings/player/player-meta/PlayerMetaTopSettings.tsx

```diff
@@ -84,6 +88,34 @@ function InspectDOM(): JSX.Element {
     )
 }
 
+function TTLWarning(): JSX.Element {
+    const { logicProps } = useValues(sessionRecordingPlayerLogic)
+    const { sessionTTLDays } = useValues(playerMetaLogic(logicProps))
+
+    if (sessionTTLDays === null || sessionTTLDays > 10) {
+        return <></>
+    }
+
+    return (
+        <div className="font-medium">
+            <LemonButton
```

Comment:
if we put a data-attr on the button then we can add an action to easily track how often it is clicked / find sessions where it is clicked